contrib/environment-to-ini/README

```diff
@@ -25,8 +25,8 @@ The environment variables should be of the form:
 Environment variables are usually restricted to a reduced character
 set "0-9A-Z_" - in order to allow the setting of sections with
 characters outside of that set, they should be escaped as following:
-"_0X2E_" for ".". The entire section and key names can be escaped as
-a UTF8 byte string if necessary. E.g. to configure:
+"_0X2E_" for "." and "_0X2D_" for "-". The entire section and key names 
+can be escaped as a UTF8 byte string if necessary. E.g. to configure:
 
 	"""
 	...
```

Comment:
At the bottom of this README it has a similar patch to this PR as suggestion, but with this PR that is no longer needed. Could you remove the part of this README at the bottom where it mentions this?